src/class_sqlThread.py

```diff
@@ -102,11 +189,20 @@ def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-s
         settingsversion = BMConfigParser().getint(
             'bitmessagesettings', 'settingsversion')
 
+        # Process SQL queries first20bytesofencryptedmessage in table inventory
+        if settingsversion == 1:
+            item = '''ALTER TABLE inventory ADD first20bytesofencryptedmessage blob DEFAULT '' '''
+            parameters = ()
+            self.cur.execute(item, parameters)
+            item = '''update settings set value=? WHERE key='version';'''
+            parameters = (2,)
+            self.cur.execute(item, parameters)
+
         # People running earlier versions of PyBitmessage do not have the
```

Comment:
lines 201-227 can be made into a separate method

Reply:
lines 227-292 can be split into 2 or 3 methods.

Reply:
Done, please check the method name.